Formula/chezmoi.rb

```diff
@@ -3,6 +3,7 @@ class Chezmoi < Formula
   homepage "https://chezmoi.io/"
   url "https://github.com/twpayne/chezmoi/archive/v1.8.0.tar.gz"
   sha256 "b55289372b0e419d9e759f3193cca366a0ae1de2bd72f523d49a0e8ca8567a47"
+  revision 1
   head "https://github.com/twpayne/chezmoi.git"
```

Comment:
If we switch to getting repo as a git repo, then we even could set `-X main.commit`

```suggestion
  url "https://github.com/twpayne/chezmoi.git",
      :tag      => "v1.8.0",
      :revision => "017a83f4055a98ac90b030d5739aa560dde239b7"
  revision 1
```